e2e/sample.config.yaml

```diff
@@ -22,8 +22,8 @@ chains:
 
 relayer:
   type: hermes # override with RELAYER_TYPE
-  image: colinaxner/hermes # override with RELAYER_IMAGE
-  tag: "v1.6.0" # override with RELAYER_TAG
+  image: ghcr.io/informalsystems/hermes # override with RELAYER_IMAGE
+  tag: "master" # override with RELAYER_TAG
```

Comment:
agree with @colin-axner , is there a non-master tag we can use that has the changes we want?

Reply:
There is no tag yet with the changes we need (they will be released in v1.7). I will close the PR then and wait until we have the tag. At least we tested that the fix works.

Reply:
I just pushed that image again to Docker Hub with the [`bef2f53` tag](https://hub.docker.com/layers/informalsystems/hermes/bef2f53/images/sha256-fb4810b31be2cab16aa6cfa415b8f62aa6f82dee75bc6a14d81f38ee4f1fa1cc?context=explore).

It's the very same one as the `master` tag you were using in this PR, but that one won't change under your feet.

Reply:
Ah sorry it's only an `arm64` image, will see if I can push the `amd64` one as well.

Reply:
All good, it's now a multi-platform image for `linux/arm64` and `linux/amd64`, the latter image is this one https://hub.docker.com/layers/informalsystems/hermes/bef2f53/images/sha256-609720ddcd9f20fac5728187e507aa6218ddd5160ffb43cb16f65c0c9e908d79

Reply:
Also available on GHCR.io now: https://github.com/informalsystems/hermes/pkgs/container/hermes/127899621?tag=bef2f53

Reply:
Thank you, @romac! Tag is updated now.